src/__tests__/key_revocation.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * @jest-environment ./build/index.js
+ */
+
+import {CeramicApi} from "@ceramicnetwork/common";
+import * as sha256 from "@stablelib/sha256";
+import * as uint8arrays from "uint8arrays";
+import KeyDidResolver from "key-did-resolver";
+import ThreeIdResolver from "@ceramicnetwork/3id-did-resolver";
+import { Resolver } from "did-resolver";
+import {DID, GeneralJWS} from "dids";
+import ThreeIdProvider from "3id-did-provider";
+import {TileDocument} from "@ceramicnetwork/stream-tile";
+import {waitForAnchor} from "../utils";
+import * as didJWT from "did-jwt";
+
+declare global {
+    const ceramic: CeramicApi;
```

Comment:
If you're only using one it should be `ceramic` not `ceramicClient`.  `ceramicClient` is *always* an http client whereas `ceramic` is sometimes an http client and sometimes a core node depending on the environment, so you get more coverage for free by using `ceramic`.

Reply:
Why it surprises you @oed ? It is the same interface for core and client.

Reply:
Now the code uses `ceramic`.

Reply:
I was surprised by Spencer's comment :)